src/components/canvas/Sphere.js

```diff
@@ -2,25 +2,39 @@ import { Suspense } from 'react'
 import { Environment, MeshDistortMaterial, Sphere } from '@react-three/drei'
 import { a, useSpring } from '@react-spring/three'
 import useStore from '@/helpers/store'
+import { A11y, useA11y, useUserPreferences } from "@react-three/a11y"
 
 const M = a(MeshDistortMaterial)
 
-const SphereComponent = () => {
-  const router = useStore((s) => s.router)
+const DarkSphere = () => {
+  const { a11yPrefersState } = useUserPreferences()
+  const a11y = useA11y()
   const { color } = useSpring({
-    color: router.route === '/box' ? '#272727' : 'black',
+    color: a11y.focus || a11y.hover ? '#272727' : 'black',
   })
+  return (
+    <Sphere
+      args={[1, 32, 32]}
+    >
+      <M distort={a11yPrefersState.prefersReducedMotion ? 0 : 0.4} color={color} />
```

Comment:
I replaced factor by distort as factor didn't seem to exist in the current version and therefore had no effect
It will leave default ( like before ) if the user is not browsing with prefer reduced motion.
Otherwise I disable the animation